tests/unit/lms/product/canvas/_plugin/course_copy_test.py

```diff
@@ -100,35 +88,24 @@ def test_find_matching_file_in_course_returns_None_if_theres_no_match(
         self, plugin, file_service
     ):
         file_service.get.return_value = factories.File(name="foo")
+        file_service.find_copied_file.return_value = None
 
         assert not plugin.find_matching_file_in_course(
             sentinel.course_id, [sentinel.file_id]
         )
 
-    def test_find_matching_file_in_course_doesnt_return_the_same_file(
```

Comment:
This testing the matching of files which is now already tested in FileService itself.